launchers/sts-server/certs/key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
maybe it would be better to add the instructions on how to generate the key/cert in the README, to avoid versioning these

Reply:
Removed and added the instructions to generate the `pfx` file